package.json

```diff
@@ -38,6 +41,11 @@
     "babel-preset-env": "^1.6.1",
     "babelify": "^8.0.0",
     "browserify": "^14.5.0",
+    "jasmine": "^2.8.0",
```

Comment:
Should we pin the versions of the npm libraries that we are using?

Reply:
They're pinned in package-lock.json - so this here is mostly just useful as a rule with something like npm-check-updates.